client/src/main/java/edu/snu/nemo/client/JobLauncher.java

```diff
@@ -72,6 +74,15 @@ private JobLauncher() {
    * @throws Exception exception on the way.
    */
   public static void main(final String[] args) throws Exception {
+    final DriverRPCServer driverRPCServer = new DriverRPCServer();
```

Comment:
Maybe add a little comment explaining that this code registers the actions for launching the DAG.